e2e/main_test.go

```diff
@@ -48,9 +48,10 @@ func TestMain(m *testing.M) {
 
 		// build and load spin operator image into cluster
 		func(ctx context.Context, _ *envconf.Config) (context.Context, error) {
-			if p := utils.RunCommand(`bash -c "cd .. && IMG=ghcr.io/spinkube/spin-operator:dev make docker-build"`); p.Err() != nil {
-
-				return ctx, fmt.Errorf(ErrFormat, p.Err(), p.Out())
+			if os.Getenv("E2E_SKIP_BUILD") == "" {
```

Comment:
gonna need to update the linter config to allow this in `e2e/`

Reply:
Missed that check 👍